Makefile

```diff
@@ -9,6 +9,7 @@ PROTO_MAPPING = MSecurity.proto=github.com/colinmarc/hdfs/v2/internal/protocol/h
 TRAVIS_TAG ?= $(shell git rev-parse HEAD)
 ARCH = $(shell go env GOOS)-$(shell go env GOARCH)
 RELEASE_NAME = gohdfs-$(TRAVIS_TAG)-$(ARCH)
+_CLIENT = $(shell kubectl get pods -l app=hdfs-client,release=my-hdfs -o name | cut -d/ -f 2)
```

Comment:
I think this sneaked in from your local testing environment?

Reply:
Oops! Thank you for pointing it out!